src/lib/common/sem.cpp

```diff
@@ -83,6 +96,42 @@ int semInit(SemRequestType _reqPolicy, int shared, int takenInitially)
 
 /* ****************************************************************************
 *
+* difftimeofday - 
+*/
+static void difftimeofday(struct timespec* endTime, struct timespec* startTime, struct timespec* diffTime)
```

Comment:
This name was picked this way because of the standard C library function 'gettimeofday'.
The name 'difftimeofday' shows that it is related to 'gettimeofday' ...

However, after creating these functions I realized gettimeofday wasn't good enough as it only gives microsconds. I found another function instead, clock_gettime, and after this change, the name difftimeofday no longer makes any sense.

Changing it for clock_difftime (and clock_addtime)